### ISSUE TITLE
Unranked -300 Elo Region South America Latam

### ISSUE THREAD

**Issue by Metalomaniac16** [RumbleMike/ValorantStreamOverlay#52]
NA Region applied. I'm Silver 1 and it doesn't show in the statistics. Last version installed

**Comment by TicianoBLTO** [comment]
me too

**Comment by BobbyESP** [comment]
Make sure u have selected NA. 2nd. Does your password contains special characters?

**Comment by Metalomaniac16** [comment]
> Make sure u have selected NA. 2nd. Does your password contains special characters?

NA already applied. It does not contain special characters.

**Comment by HolidayLNX** [comment]
> NA Region applied. I'm Silver 1 and it doesn't show in the statistics. Last version installed

Have you played 3 or more ranked games recently?

**Comment by Metalomaniac16** [comment]
> > NA Region applied. I'm Silver 1 and it doesn't show in the statistics. Last version installed
> 
> Have you played 3 or more ranked games recently?

Yes. In fact i played a ranked game after installing it precisely to discard info update issues.

**Comment by pasya1912** [comment]
i think this error come from riot itself. i tried to create my own program with the api provided by @RumbleMike . and i got the same result...
